go.mod

```diff
@@ -184,6 +184,7 @@ require (
 	github.com/robfig/cron v1.2.0 // indirect
 	github.com/russross/blackfriday v1.6.0 // indirect
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
+	github.com/second-state/WasmEdge-go v0.13.0 // indirect
```

Comment:
Is this indirect or direct?
Try using `go mod tidy`

Reply:
Resolved in d87520b